tests/src/com.hazelcast/hazelcast/5.2.1/src/test/resources/META-INF/native-image/javax.cache/cache-api/1.1.1/resource-config.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
This looks like that the Hazelcast support needs `javax.cache`. I guess without those files the tests won't run. If this is the case, these files should be provided as a separate PR to support the `javax.cache:cache-api` dependencies.

Reply:
- I did open it https://github.com/oracle/graalvm-reachability-metadata/issues/136 .
- However, I have no idea how to change the content of `serialization-config. json` of `javax.cache:cache-api:1.1.1` to the `graalvm reachability metadata conditional` form. I added that part manually. The error log of GraalVM gives little information about serialization.

Reply:
Ah, thanks. I missed that.

Reply:
> Would this help to make it conditional?
> 
> https://docs.oracle.com/en/graalvm/enterprise/22/docs/reference-manual/native-image/metadata/

- I've seen the docs before. The reason why I need to manually add this part of metadata is that when I use GraalVM Trace Agent, even if I configure the conditional mode for the `javax.cache.*` package, I cannot collect the GraalVM reachability metadata related to serialization. I have to turn on the standard mode to manually find the serialization metadata that I really need, which makes me don't know where to start. 

- The key point is that when the metadata related to serialization is missing, the error log of GraalVM CE 22.3 does not prompt what the context of the required class is, which is very different from the prompt when the metadata of reflection is missing.

Reply:
OK, these both seem like bugs, thanks for pointing it out. Just to clarify, by context you mean the stack trace where serialization should happen? 

Would it be possible to open a [ticket](https://github.com/oracle/graal/issues) with a command to reproduce?

Reply:
- If you think this is a bug, I will open a new issue at https://github.com/oracle/graal and provide a reproducible example. 

- So far, I have only found that the serialization metadata of `javax.cache.*` is required in the unit test of hazelcast. I am not sure whether this is related to https://github.com/oracle/graalvm-reachability-metadata/pull/169 #issuecomment-1378981023 . hazelcast's GraalVM reachability metadata differs significantly from other JSR-107 implementations, such as Ehcache and Caffeine.

Reply:
@vjovanov I misunderstood the Log, I dealt with this at https://github.com/oracle/graalvm-reachability-metadata/pull/169#issuecomment-1382704703, there doesn't seem to be anything to handle on the GraalVM side.